docs/_posts/2019-05-01-compose-installation-guide.md

```diff
@@ -0,0 +1,487 @@
+`/var/lib/mysql` inside the MySQL container, and to `/storage/miso/files/`
+inside the webapp container.
+
+### Docker volumes
+
+Docker volumes can be used for testing and semi-permanent demonstrations, when
+you don't want to lose all of your data between restarts.
+
+First ensure that the docker-compose file is working
+correctly before adding persistent storage. Otherwise you risk corrupting your
+database.
+
+Then modify the docker-compose.yml file with the following:
+
+``` yaml
+...
+services:
+  db:
+    volumes:
+      - "misolims_db:/var/lib/mysql"
+  webapp:
+    volumes:
+      - "misolims_files:/storage/miso/files/"
+...
+volumes:
+  misolims_db:
+  misolims_files:
+```
+
+### Filesystem mount
+
+For permanent, production use, we recommend binding a location on the host
+filesystem for the database and file storage.
+
+First ensure that the docker-compose file is working correctly before adding
+persistent storage. Otherwise you risk corrupting your database.
+
+Then modify the docker-compose file with the following, where `/path/to` and
+subdirs are locations on the host filesystem:
+
+``` yaml
+...
+services:
+  db:
+    volumes:
+      - "/path/to/mysql/db:/var/lib/mysql"
+  webapp:
+    volumes:
+      - "/path/to/files:/storage/miso/files/"
+```
+
+## Pre-loading data with Flyway
+
+Starting MISO with an entirely empty database will not provide full functions.
+The demo database starts with some pre-loaded information that may or may not be
+useful: see the SQL files
+[here](https://github.com/miso-lims/miso-lims/tree/master/.docker/detailed_sample_config).
+
+We recommend including the V0792 migration to
+clean out the database and modifying V0793 as appropriate for your institution.
+All SQL migrations that conform with Flyway's `V####__name.sql` naming scheme
+will be automatically run in numerical order when mounted at `/flyway/sql`
+inside the container.
+
+
+``` yaml
+...
+services:
+  flyway:
+    volumes:
+      - "./detailed_sample_config/V0792__drop-inconvenient-rows.sql:/flyway/sql/V0792__drop-inconvenient-rows.sql:ro"
+      - "./V1000__institution-custom.sql:/flyway/sql/V1000__institution-custom.sql:ro"
+...
+```
+
+See the
+[administrator's manual](http://miso-lims.github.io/miso-lims/adm/admin-guide.html)
+for more information on the types of data that can be added at this stage.
+
+## Modifying the miso.properties file
+
+The
+[miso.properties](https://github.com/miso-lims/miso-lims/blob/master/miso-web/src/main/resources/internal/miso.properties)
+file has plenty of options to configure for site-specific use. Some important
+options include:
+* `miso.detailed.sample.enabled` : turns detailed sample mode on and off
+* `miso.instanceName` : displays an instance name in the header on the webapp
+  (to distinguish prod and stage, for example).
+* `miso.naming.scheme` : turns on/off various validation options for naming
+  samples. The most complete is `oicr`.
```

Comment:
Why recommend `oicr`? Unlike the `default` naming scheme, it doesn't allow configuration of components and does require our sample/library alias validators. (It might be worth linking to the ["Naming Schemes (updating $CATALINA_HOME/conf/Catalina/localhost/miso.properties)"](https://miso-lims.github.io/miso-lims/adm/installation-guide.html) section here, if possible)

Reply:
I've changed it to 'most rigorous'.